library-best-practices/JLBP-11.md

```diff
@@ -0,0 +1,7 @@
+[JLBP-11] Frequent release cadence for non-breaking changes
```

Comment:
Phrase imperatively: Release non-breaking changes frequently

Reply:
Hmm I realized that my original title (and thus your variant) has the wrong implications - it implies that libraries should just keep adding things, regardless of their value. I don't actually care how frequently libraries push out new features - it could be never for all I care.

The more important thing is making releases when dependencies make new releases. How about: "Stay up to date with compatible dependencies"?